src/config.rs

```diff
@@ -378,6 +378,111 @@ impl de::Deserialize for ActionWrapper {
     }
 }
 
+struct CommandWrapper {
+    program: String,
+    args: Vec<String>,
+}
+
+impl de::Deserialize for CommandWrapper {
```

Comment:
So, this whole block can be implemented automatically in this case. Just need to put a `#[derive(Deserialize)]` above `struct CommandWrapper`.